EIPS/eip-draft_universal_router.md

```diff
@@ -0,0 +1,282 @@
+---
+eip: <to be assigned>
+title: Universal Router Contract
+description: Universal router contract designed for token allowance that eliminates all `approve` transactions in the future.
+author: Zergity <zergity@gmail.com>
+discussions-to: <URL>
+status: Draft
+type: Standards Track
+category: ERC
+created: 2022-12-12
+requires: EIP20, EIP721, EIP1155
+---
+
+## Abstract
+
+A universal router contract that executes transactions with the following steps:
```

Comment:
You should also expand on what a router contract is, what field this is relevant for (EIP-20 tokens, allowances, etc.)